int8 mode
int8 tar1
int8 tar2
int8 if_takeoff 

